src/GitHub/Data/PullRequests.hs

```diff
@@ -180,8 +184,22 @@ instance Binary PullRequestReference
 -- JSON instances
 -------------------------------------------------------------------------------
 
+-- | Helper function, reads either the "assignee" OR "assigneed" OR
+-- both from a JSON object.
+getAssignees :: Object -> Parser (Vector SimpleUser)
```

Comment:
[this example](https://developer.github.com/v3/pulls/#list-pull-requests) has an `assignee` key but no `assignees` keys.